package.json

```diff
@@ -1,32 +1,34 @@
 {
   "name": "livepeer",
-  "version": "3.0.2",
```

Comment:
What do you think about this being a major version bump? Since we now have a new peer dependency, Zod, that developers will need to use.

Reply:
Sounds good! I will bump it to 3.1.0